Userland/Utilities/ls.cpp

```diff
@@ -54,7 +54,9 @@ enum class IndicatorStyle {
     Directory = 1 << 0,
     Executable = 1 << 1,
     SymbolicLink = 1 << 2,
-    Classify = Directory | Executable | SymbolicLink
+    Pipe = 1 << 3,
+    Sock = 1 << 4,
```

Comment:
We try to avoid unnecessary abbreviations in identifiers, please name it `Socket`.